src/octopeer-github/test/ElementSelectionBehaviour/SelectButtonsElementSelectionBehaviourTest.ts

```diff
@@ -1,26 +1,153 @@
+/* tslint:disable */
```

Comment:
Am I allowed to be more specific here? :innocent:
Please change it to: /* tslint:disable:max-line-length */
Same holds for the enable, at the bottom of this giant reference block.